pkg/sql/catalog/descpb/constraint.go

```diff
@@ -125,3 +125,22 @@ func (c *ConstraintDetail) GetConstraintName() string {
 	}
 	return ""
 }
+
+// SetConstraintName sets correct constraint name base on the constraint
+// type.
+func (c *ConstraintDetail) SetConstraintName(name string) {
+	switch c.Kind {
+	case ConstraintTypePK:
+		c.Index.Name = name
+	case ConstraintTypeUnique:
+		if c.Index != nil {
+			c.Index.Name = name
+		} else {
+			c.UniqueWithoutIndexConstraint.Name = name
+		}
+	case ConstraintTypeFK:
+		c.FK.Name = name
+	case ConstraintTypeCheck:
+		c.CheckConstraint.Name = name
+	}
+}
```

Comment:
As it turns out, `ConstraintDetail` is now no longer used anywhere and can be removed entirely (which I forgot to do)

Reply:
yeah, I've deleted both methods (GetConstraintName and SetConstraintName) bc they are not used anywhere.